### PR TITLE
Credits-Page

### PR DESCRIPTION
**Pull request by jlntrt**

### CODE REVIEW

**Comment by sphinxc0re** [comment]
How about adding the Github handle as well? @zaolin @debao84 @schutzle

**Comment by sphinxc0re** [comment]
Also, could you add a screenshot of the current status?

**Comment by jlntrt** [comment]
Sure. Github handles are added. Current status:
screen shot 2016-12-27 at 16 54 10

Pretty simple as mentioned.

File: app/views/frontend/credits.html.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+= render 'top_bar'
+.container
+  .row
+    .col.s12.m6
+      .content
+        - @members.each do |name, media|
+          .card.blue-grey.darken-1
```
Comment (sphinxc0re) [inline app/views/frontend/credits.html.haml]: grey.darken-2 or 1 would be better

**Review by sphinxc0re — CHANGES_REQUESTED** [review]

**Comment by jlntrt** [comment]
Like that: 
screen shot 2016-12-27 at 17 27 32

**Review by sphinxc0re — APPROVED** [review]

**Comment by jlntrt** [comment]
@sphinxc0re @zaolin can I merge this branch into `master`?

**Comment by sphinxc0re** [comment]
Done :grin: good job!

**Pull request merged**